src/components/header/header.tsx

```diff
@@ -4,17 +4,20 @@ import { up } from '../breakpoint/breakpoint';
 import styled from 'styled-components';
 import BurgerMenu from '../icons/burger-menu';
 import { NavigationFlyout } from './menu-flyout';
+import { FluidObject } from 'gatsby-image';
+import { HeroImage } from './hero-image';
+import { GRID_GAP_DESKTOP, GRID_GAP_MOBILE } from '../grid/grid';
 
 export const HEADER_HEIGHT = '65px';
 
-const StyledHeader = styled.header<{ light: boolean }>`
+const StyledHeader = styled.header<{ light: number }>`
   height: ${HEADER_HEIGHT};
   display: flex;
   justify-content: space-between;
   align-items: center;
   padding: 0 16px;
   border-bottom: ${(props) =>
-    props.light
+    props.light === 1
```

Comment:
I'm curious why the change to a number ? I didn't expect it to be a number.

Reply:
If you use a styled component with the boolean you get this error: https://github.com/styled-components/styled-components/issues/1198.

Interestingly, the error only appears with the `SiteTitle` component in this file, and not the `StyledHeader` where I did this change 🤷 . To have a bit more consistency in this file, I also changed the prop here to a number.